privacyidea/lib/utils/export.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+log = logging.getLogger(__name__)
+
+EXPORT_FUNCTIONS = {}
+IMPORT_FUNCTIONS = {}
+
+
+def register_export(name=None):
+    def wrapped(func):
+        # TODO: this doesn't work since the logging configuration is not
+        #       initialized during imports
+        # log.debug('Registering export function {0!s} with name {1!s}'.format(func, name))
```

Comment:
Please check if you want to keep this many TODOs.

Reply:
Done in ef40697f